main.go

```diff
@@ -3,9 +3,9 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/davrodpin/ovsdbviz/graphviz"
-	"github.com/davrodpin/ovsdbviz/ovsdb"
 	"os"
+	"ovsdbviz/graphviz"
+	"ovsdbviz/ovsdb"
```

Comment:
I think this change needs to be reverted. Both imports should include the whole github address.